### PR TITLE
New IBM datastage cheat sheet

### PR DESCRIPTION
**Pull request by mohan08p**
IA page : https://duck.co/ia/view/ibm_datastage_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/datastage.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    "id": "ibm_datastage_cheat_sheet",
+    "name": "IBM DataStage",
+    "description":"It is an ETL tool uses a graphical notation to construct data integration solutions and is available in various versions such as the Server Edition, the Enterprise Edition, and the MVS Edition",
+    "metadata": {
+        "sourceName": "Nuts & Bolts of DataStage",
+        "sourceUrl": "http://datastage4you.blogspot.in/2014/10/vi-cheat-sheet.html"
+    },
+    "template_type": "terminal",
+    "section_order": [
+        "Cursor movement",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/datastage.json]: @mohan08p What about "Cursor Movement"?

File: share/goodie/cheat_sheets/json/datastage.json [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        "Insert Mode - Inserting/Appending text",
+        "Editing",
+        "Marking text (visual mode)",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/datastage.json]: @mohan08p Marking Text

File: share/goodie/cheat_sheets/json/datastage.json [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+        "Visual commands",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/datastage.json]: Visual Commands

File: share/goodie/cheat_sheets/json/datastage.json [hunk 4]
```diff
@@ -0,0 +1,257 @@
+{
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/datastage.json]: @mohan08p please change the filename to `ibm-datastage.json`

File: share/goodie/cheat_sheets/json/ibm-datastage.json [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+            "key": ":sp filename"
+        }, {
+            "val": "Split windows",
+            "key": "ctrl+ws"
+        }, {
+            "val": "Switch between windows",
+            "key": "ctrl+ww"
+        }, {
+            "val": "Quit a window",
+            "key": "ctrl+wq"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/ibm-datastage.json]: I'm not a huge fan of the mixed content here -- we should try and keep shortcuts out if it's mostly commands

**Comment by moollaza** [comment]
@mohan08p Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@mohan08p this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**